app/views/handlers/confirmation_email.py

```diff
@@ -41,11 +41,11 @@ def __init__(
         self._serialised_email = serialised_email
 
     @property
-    def page_title(self):
+    def page_title(self) -> str | LazyString | None:
```

Comment:
Do you need `LazyString` when you got `str`?

Reply:
When we were going through this, mypy didn't like that lazy_gettext("Confirmation email") returned a LazyString hence why we included it. Re-visiting it now, it doesn't throw and error so will remove it